lib/cisco_node_utils/cmd_ref/interface_port_channel.yaml

```diff
@@ -0,0 +1,56 @@
+# interface_port_channel
+---
+_template:
+  config_set: ["interface %s"]
+  config_get_token: '/^interface %s$/i'
+  config_get: "show running interface all"
+
+create:
+  config_set: "interface %s"
+
+destroy:
+  config_set: "no interface %s"
+
+lacp_graceful_convergence:
+  kind: boolean
+  auto_default: false
+  config_get_token_append: ['/^lacp graceful.convergence$/']
+  config_set_append: ["%s lacp graceful-convergence"]
+  default_value: true
+
+lacp_max_bundle:
+  kind: int
+  config_get_token_append: ['/^lacp max.bundle (\d+)$/']
+  config_set_append: ["%s lacp max-bundle %s"]
+  /N3K/:
+    default_value: 32
+  /N9K/:
+    default_value: 32
+  else:
+    default_value: 16
+
+lacp_min_links:
+  kind: int
+  config_get_token_append: ['/^lacp min.links (\d+)$/']
+  config_set_append: ["%s lacp min-links %s"]
+  default_value: 1
+
+lacp_suspend_individual:
+  kind: boolean
+  auto_default: false
+  config_get_token_append: ['/^lacp suspend.individual$/']
+  config_set_append: ["%s lacp suspend-individual"]
+  default_value: true
+
+port_hash_distribution:
+  _exclude: [/N6K/, /N5K/]
```

Comment:
We've started to remove the `K`'s from the match because N7K will miss N77, and N3K will miss some N3 platforms, so just do:

`  _exclude: [/N5/, /N6/]`

Ditto for the other yaml file.